tests/test_split_flatpak_repo.py

```diff
@@ -221,18 +243,52 @@ def create_flatpak_commits(self):
         flatpak_refs.update(self.random_commit(
             self.os_repo,
             ('flathub:app/org.gnome.Totem/x86_64/stable',
-             'deploy/app/org.gnome.Totem/x86_64/stable')
+             'deploy/app/org.gnome.Totem/x86_64/stable'),
+            flatpak_metadata=dedent("""\
```

Comment:
I always forget this trailing-slash syntax and instead stick a `.strip()` on the end. Nice.